CHANGES.md

```diff
@@ -1,3 +1,144 @@
+## Version 3.0.0-RC4 (August 25, 2019)
+
+Last release before `3.0.0` which will be released as soon as Cats-Effect 2.0.0 is available. At the time of writing release notes,
+it is only waiting on Cats 2.0.0 which is about to release next RC which will become stable version if no critical issues are discovered.
+
+Any other development for this release is now frozen except if we discover critical bugs like memory leaks. All other
+changes will target `3.1.0`. 
+
+`3.0.0` will be binary and source compatible with `3.0.0-RC4` for Scala 2.12. Monix itself will be also binary compatible
+for 2.11 but it will have a dependency on Cats-Effect which is not. [See Cats-Effect release notes](https://github.com/typelevel/cats-effect/releases/tag/v2.0.0-RC1) for more details.
+
+We're very sorry for such a long wait. We wish we could release `3.0.0` already but if we released now, we would have to 
```

Comment:
Don't apologize so much. It is what it is. It's not like we're paid for it.

Reply:
Saying that we need to synchronize with Cats and Cats-Effect 2.0 and that we are in feature freeze ... is IMO enough.

Reply:
So to expand on this point — we are not in under any contractual obligation, we are not getting paid for it and feeling bad about delaying a final release in an open source is a recipe for burnout, since it stops being fun.

I think we've got a solid release and that it will happen along with Cats-Effect 2.0, we worked hard on it, so that's nothing to apologize for.

This is me giving unsolicited advice, given that I experienced burnout before 🙂

Reply:
Thanks, that's a good advice.

Updated it